### PR TITLE
bpo-32610: Fix asyncio.all_tasks() to return only pending tasks.

### PR DESCRIPTION
**Pull request by 1st1**
https://bugs.python.org/issue32610

### CODE REVIEW

File: Lib/asyncio/tasks.py [hunk 1]
```diff
@@ -33,6 +33,16 @@ def current_task(loop=None):
 
 def all_tasks(loop=None):
     """Return a set of all tasks for the loop."""
+    if loop is None:
+        loop = events.get_event_loop()
```
Comment (asvetlov) [inline Lib/asyncio/tasks.py]: `get_running_loop()` maybe?
If `all_tasks()` is not the same as `Task.all_tasks()` anymore...

**Review by asvetlov — COMMENTED** [review]
Documentation should reflect the difference

File: Lib/asyncio/tasks.py (hunk 1 above)
Comment (1st1) [reply Lib/asyncio/tasks.py]: I'd go for running, yes; it was my original request.  Let me update the PR.

**Comment by 1st1** [comment]
@asvetlov updated

**Review by asvetlov — APPROVED** [review]

**Pull request merged**